RUFAS/data_validator.py

```diff
@@ -1606,3 +1606,120 @@ def extract_value_by_key_list(self, data: list[Any] | dict[str, Any], variable_p
             else:
                 raise KeyError(f"There is an error at key {key} in the path {variable_path}")
         return data
+
+
+class CrossValidator:
+    def __init__(self) -> None:
+        self._alias_pool: dict[str, Any] = {}
+        self._event_logs: list[dict[str, str | dict[str, str]]] = []
```

Comment:
What is the `_event_logs` list for (i.e. is there a reason we can't just log stuff directly to OM)?